concent_api/core/models.py

```diff
@@ -712,7 +734,7 @@ def clean(self) -> None:
 
 
 class DepositClaim(Model):
-    subtask = ForeignKey(Subtask, blank=True, null=True)
+    subtask_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH, blank=True, null=True)
```

Comment:
Actually we have everywhere `blank = True, null = True`. I think it is ok, as it all depends on the use case. I think that `blank = True, null = False` is used more in web development, when you often print the value directly to template. We don't do this, and it is better in my opinion to compare against `None` than empty string.

Reply:
> OK. But are you sure we're using this consistently?

Yes, in every field which use those two. And I think it works best for our case, because we don't allow users to edit data, we don't do it through admin panel etc. Actually when I have checked it, in all those places we could have `blank=False, null=True` rather than the opposite.
And I think this recommendation which you have quoted is common for web development.

Reply:
I mean that it's common not only in web development. It's simply common in Django.